src/main/java/competition/auto_programs/ScoreCubeHighThenExitCommunityAndBalance.java

```diff
@@ -0,0 +1,148 @@
+import competition.subsystems.claw.CloseClawCommand;
+import competition.subsystems.drive.commands.AutoBalanceCommand;
+import competition.subsystems.drive.commands.BrakeCommand;
+import competition.subsystems.drive.commands.SwerveSimpleTrajectoryCommand;
+import competition.subsystems.drive.commands.VelocityMaintainerCommand;
+import competition.subsystems.drive.commands.XbotSwervePoint;
+import competition.subsystems.pose.PoseSubsystem;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScoreCubeHighThenExitCommunityAndBalance extends SequentialCommandGroup{
+    @Inject
+    public ScoreCubeHighThenExitCommunityAndBalance(
+            PoseSubsystem pose,
+            ScoreCubeHighCommandGroup scoreCubeHigh,
+            SimpleXZRouterCommand retractArm,
+            CloseClawCommand closeClaw,
+            SwerveSimpleTrajectoryCommand turnTowardsField,
+            SwerveSimpleTrajectoryCommand exitCommunity,
+            SwerveSimpleTrajectoryCommand turnTowardsCommunity,
+            SwerveSimpleTrajectoryCommand mantleChargePlate,
+            AutoBalanceCommand autoBalance,
+            VelocityMaintainerCommand velocityMaintainer,
+            BrakeCommand brake) {
+
+        // Force set our current position - facing the grid station, ready to score our cone.
+        var forcePosition = new InstantCommand(
+                () -> {
+                    var translation =
+                            AutoLandmarks.convertBlueToRedIfNeeded(AutoLandmarks.blueScoringPositionFive)
+                                    .getTranslation().times(1.0 / PoseSubsystem.INCHES_IN_A_METER);
+                    var rotation = pose.rotateAngleBasedOnAlliance(Rotation2d.fromDegrees(-180));
+                    pose.setCurrentPoseInMeters(new Pose2d(translation, rotation));
+                }
+        );
+        this.addCommands(forcePosition);
+
+        // Score high
+        this.addCommands(scoreCubeHigh);
+
+        // Get the arm back safe
+        retractArm.setKeyPointFromKeyArmPosition(
+                UnifiedArmSubsystem.KeyArmPosition.PrepareToAcquireFromCollector,
+                UnifiedArmSubsystem.RobotFacing.Forward);
+
+        var retractArmAndCloseClaw = new ParallelCommandGroup(
+                retractArm,
+                new WaitCommand(1.0).andThen(closeClaw)
+        ).withTimeout(2.0);
+
+        this.addCommands(retractArmAndCloseClaw);
+
+        turnTowardsField.setMaxPower(1.0);
+        turnTowardsField.setMaxTurningPower(0.5);
+        turnTowardsField.setKeyPointsProvider(
```

Comment:
turnTowardsField and exitCommunity can be combined (one command with multiple "points" visited)